notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2216-plots.ipynb

```diff
@@ -0,0 +1,1166 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "dd54f85e-a3c9-4eb5-b37f-533d27ececa3",
+   "metadata": {},
+   "source": [
+    "# [LVV-T2216] Integrated Slew and Tracking Test - Data Analysis\n",
+    "\n",
+    "This notebook is used to analyse the data colleted when running the [LVV-T2216] test case during System Spread Integration Tests on Level 3.  \n",
+    "   \n",
+    "**Requirements:**\n",
+    " - You should have run this test case and record the time stamps on each step.\n",
+    " \n",
+    "Upon completion, save the notebook and its output as a pdf file to be attached to the test execution in JIRA. \n",
+    "\n",
+    "**Make sure you run this notebook on TTS before running at the summit.**\n",
+    "\n",
+    "Please, see the [README] file for the requirements to run this notebook.\n",
+    "\n",
+    "[LVV-T2216]: https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T2216\n",
+    "[README]: https://github.com/lsst-sitcom/notebooks_vandv/blob/develop/README.md"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8362b636-22d3-409b-9bd0-ddab4294ec82",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_case = \"LVV-T2216\"\n",
+    "test_exec = \"LVV-EXXXX\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8caef44f-e7da-4abd-8160-a5776f67b883",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from astropy.time import Time\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "from lsst.sitcom import vandv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2df2cfb2-4fe4-4ce2-a963-359401eed120",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = vandv.efd.create_efd_client()\n",
+    "\n",
+    "exec_info = vandv.ExecutionInfo()\n",
+    "print(exec_info)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5c69a6f-915f-4963-87de-9e5a6b5ca8ff",
+   "metadata": {},
+   "source": [
+    "## Slewing without LUT corrections\n",
+    "\n",
+    "In theory, the LUT is always being applied to M1M3, M2, and the Hexapods.  \n",
+    "This section considers that the elevation angle used for LUT calculations comes from the inclinometer and is always 90 deg.  \n",
```

Comment:
If LUT uses inclinometer (always at 90deg), then the force actuators wouldn't change with elevation, right?

Reply:
Yes, but this is true only on Level 3. On TTS, the telescope is actually moving, so the inclinometer values are simulated.